<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="rnd_0">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>120</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border: 3px solid black;
border-radius: 75px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="rnd_1">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>120</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border: 3px solid black;
border-radius: 75px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="rnd_2">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>120</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border: 3px solid black;
border-radius: 75px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="rnd_5">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>300</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border: 3px solid black;
border-radius: 75px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="rnd_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>300</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border: 3px solid black;
border-radius: 75px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="rnd_4">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>300</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border: 3px solid black;
border-radius: 75px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="rnd_9">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>470</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border: 3px solid black;
border-radius: 75px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="rnd_7">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>470</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border: 3px solid black;
border-radius: 75px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="rnd_8">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>470</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border: 3px solid black;
border-radius: 75px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="rnd_6">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>300</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border: 3px solid black;
border-radius: 75px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="result">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>0</y>
      <width>310</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Upheaval Pro</family>
      <pointsize>32</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 127);
border-bottom-left-radius: 30px;</string>
    </property>
    <property name="text">
     <string>Результат: 0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_back">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>170</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Upheaval Pro</family>
      <pointsize>32</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 255, 160);
border-bottom-right-radius: 30px</string>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>720</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Yandex Lyceum/background_for_menu.png</pixmap>
    </property>
   </widget>
   <zorder>background</zorder>
   <zorder>rnd_0</zorder>
   <zorder>rnd_1</zorder>
   <zorder>rnd_2</zorder>
   <zorder>rnd_5</zorder>
   <zorder>rnd_3</zorder>
   <zorder>rnd_4</zorder>
   <zorder>rnd_9</zorder>
   <zorder>rnd_7</zorder>
   <zorder>rnd_8</zorder>
   <zorder>rnd_6</zorder>
   <zorder>result</zorder>
   <zorder>btn_back</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
